docs/miscellaneous/create-dns-stamp.md

```diff
@@ -24,21 +24,22 @@ Types of Secure DNS include `DNS-over-HTTPS (DoH)`, `DNS-over-QUIC (DoQ)`, and `
 
 3. Fill in the necessary fields:
     - **IP address**:
+    Enter the IP address of the DNS server. If you are using the DoT or DoQ protocol, make sure that you have specified the appropriate port as well.
+
     :::note
 
     This field is optional and should be used with caution: using this option may disrupt the Internet on IPv6-only networks.
 
     :::
-
-    Enter the IP address of the DNS server. If you are using the DoT or DoQ protocol, make sure that you have specified the appropriate port as well.
     - **Hashes**:
+    Enter the SHA256 digest of one of the TBS certificates found in the validation chain. If the DNS server you are using provides a ready-made hash, find and copy it. Otherwise, you can obtain it by following the instructions in the ["Obtaining the Certificate Hash"](###obtaining-the-certificate-hash) section.
```

Comment:
```suggestion
    Enter the SHA256 digest of one of the TBS certificates found in the validation chain. If the DNS server you are using provides a ready-made hash, find and copy it. Otherwise, you can obtain it by following the instructions in the [*Obtaining the Certificate Hash*](###obtaining-the-certificate-hash) section.
```

Reply:
названия разделов в англ выделяем курсивом